app/admin/models/config/menu_option_values_model.php

```diff
@@ -16,6 +16,11 @@
             'label' => 'lang:admin::lang.menu_options.label_option_price',
             'type' => 'currency',
         ],
+        'stock_qty' => [
+            'label' => 'lang:admin::lang.menu_options.label_stock_qty',
+            'type' => 'stockeditor',
+            'span' => 'right',
+        ],
         'allergens' => [
```

Comment:
Can we make allergens hide when there are none?

Reply:
How do we go about doing so?

Reply:
Could do a query in the config file I guess. Other option is to handle it in the relation field type with a new attribute so it doesn't break compatibility where its currently used?

Reply:
Yea we can add a new property on the repeater widget, something like the List widget's invisible columns?

Reply:
Probably better on the relation widget - "hideWhenHasNoOptions" or something like that. Then when true, check the length of $clonedFormField and hide the widget if 0 ?